pkg/apis/core/v1alpha1/defaults.go

```diff
@@ -321,6 +321,34 @@ func SetDefaults_Worker(obj *Worker) {
 	}
 }
 
+// SetDefaults_ClusterAutoscaler sets default values for ClusterAutoscaler object.
+func SetDefaults_ClusterAutoscaler(obj *ClusterAutoscaler) {
+	if obj.ScaleDownDelayAfterAdd == nil {
+		obj.ScaleDownDelayAfterAdd = &metav1.Duration{Duration: 1 * time.Hour}
+	}
+	if obj.ScaleDownDelayAfterDelete == nil {
+		obj.ScaleDownDelayAfterDelete = &metav1.Duration{Duration: 0}
+	}
+	if obj.ScaleDownDelayAfterFailure == nil {
+		obj.ScaleDownDelayAfterFailure = &metav1.Duration{Duration: 3 * time.Minute}
+	}
+	if obj.ScaleDownUnneededTime == nil {
+		obj.ScaleDownUnneededTime = &metav1.Duration{Duration: 30 * time.Minute}
+	}
+	if obj.ScaleDownUtilizationThreshold == nil {
+		obj.ScaleDownUtilizationThreshold = pointer.Float64(0.5)
```

Comment:
For consistency, add the default value in the documentation.

Reply:
Done [here](https://github.com/gardener/gardener/pull/4508/commits/ed626583966d98e816e697c1b4780373f0a3739d#diff-63c59db8a50bf6d0d486e8ae5b3ec2b334d192fe2f1bbe8d27c0c6742eeb51c3R2629)